Document/0x06a-Platform-Overview.md

```diff
@@ -131,27 +131,28 @@ A language.lproj folder is defined for each language that the application suppor
 
 - `/var/mobile/Containers/Bundle/Application/[UUID]/Application.app` contains the previously mentioned application.app data and stores the static content as well as the ARM compiled binary of the application. The content of this folder will be used to validate the code signature.
-- `/var/mobile/Containers/Data/Application/[UUID]/Documents` contains all the data stored for the application itself. The creation of this data is initiated by the application’s end user.
-- `/var/mobile/Containers/Data/Application/[UUID]/Library` contains files necessary for the application e.g. caches, preferences, cookies, property list (plist) configuration files, etc.
-- `/var/mobile/Containers/Data/Application/[UUID]/Temp` contains temporary files which do not need persistence in between application launches.
+- `/var/mobile/Containers/Data/Application/[UUID]/Documents` contains all the data stored for the application itself. The creation of this data is initiated by the application end user.
+- `/var/mobile/Containers/Data/Application/[UUID]/Library` contains the files used by the application e.g. caches, preferences, cookies, property list (plist) configuration files, etc.
+- `/var/mobile/Containers/Data/Application/[UUID]/Temp` contains temporary files which are not needed between application launches.
 
-The following figure represents the application’s folder structure:
+The following figure represents the application folder structure:
 
 <img src="/Images/Chapters/0x06a/iOS_Folder_Structure.png" width="500px"/>
 - *iOS App Folder Structure*
 
 #### The Installation Process
 
-Different methods exist to install an IPA package on the device. The easiest solution is to use iTunes, which is the default media player from Apple. iTunes is available for macOS as well as for Windows. iTunes allows you to download applications through the App Store, after which you can install/synchronize them to an iOS device. You can also use [iTunes to install an IPA file to a device](https://www.youtube.com/watch?v=nNn85Qvznug "How to install an app via iTunes").
+Different methods exist to install an IPA package on an iOS device. The easiest solution is to use iTunes, which is the default media player from Apple. iTunes is available for macOS as well as for Windows. iTunes allows users to download applications through the App Store and install them to the iOS device. One can also use [iTunes to install an IPA file to a device](https://www.youtube.com/watch?v=nNn85Qvznug "How to install an app via iTunes").

```

Comment:
Please see the style guide https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md address the reader in the second person.